R/build.r

```diff
@@ -434,6 +434,12 @@ build_site_local <- function(pkg = ".",
   build_articles(pkg, lazy = lazy, override = override, preview = FALSE)
   build_tutorials(pkg, override = override, preview = FALSE)
   build_news(pkg, override = override, preview = FALSE)
+  build_sitemap(pkg)
```

Comment:
If the sitemap is to include all paths it needs to be built at this stage, and `build_docsearch_json()` needs the sitemap.